File: java.se2/src/test/java/com/mycompany/sample/tests/pages/RegisterPage.java
package com.mycompany.sample.tests.pages;

import com.mycompany.sample.tests.Util.WebBrowser;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

/**
 * Created by IntelliJ IDEA.
 * UserUtil: aaron.backer
 * Date: 3/17/12
 * Time: 3:20 PM
 * To change this template use File | Settings | File Templates.
 */
public class RegisterPage extends BasePage {


    public RegisterPage(WebBrowser browser) {
        super(browser);
    }

    public WebElement txtUsername() {
        return _browser.element(By.id("UserName"));
    }
    
    public WebElement txtPassword() {
        return _browser.element(By.id("Password"));
    }
    
    public WebElement txtConfirmPassword() {
        return _browser.element(By.id("ConfirmPassword"));
    }
    
    public WebElement txtEmail() {
        return _browser.element(By.id("Email"));
    }

    public WebElement btnRegister() {
        return _browser.element(By.xpath(".//input[@value='Register']"));
    }
}

File: SeBCC-2014/SeBCC/Page/RegisterPage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using OpenQA.Selenium;

namespace SeBCC
{
    public class RegisterPage : AbstractPage

    {
        public RegisterPage(Browser browser) : base(browser)
        {
        }

        public IWebElement Username()
        {
            return _browser.Find(By.Id("UserName"));
        }

        public IWebElement Email()
        {
            return _browser.Find(By.Id("Email"));
        }

        public IWebElement Password()
        {
            return _browser.Find(By.Id("Password"));
        }

        public IWebElement ConfirmPassword()
        {
            return _browser.Find(By.Id("ConfirmPassword"));
        }

        public IWebElement RegisterBtn()
        {
            return _browser.Find(By.XPath(".//input[@value='Register']"));
        }
    }
}
